Modules/Core/TestKernel/test/itkTestingComparisonImageFilterGTest.cxx

```diff
@@ -0,0 +1,108 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         https://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+
+#include "itkTestingComparisonImageFilter.h"
+#include "itkImage.h"
+#include "itkVectorImage.h"
+
+#include "itkGTest.h"
+
+
+// Write a Google test to check the class's run time type information
+TEST(itkTestingComparisonImageFilterTest, CheckRunTimeTypeInformation)
```

Comment:
Minor suggestion: I think you may remove the first part of the sentence, "Write a Google test to ". Because you did actually do that already: writing such a test!

Reply:
O, That was the prompt use to generate the test function. 😀